src/server/table_stats.h

```diff
@@ -1,187 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-#pragma once
-
-struct table_stats
-{
-    table_stats(const std::string &app_name) : app_name(app_name) {}
-
-    double get_total_read_qps() const
-    {
-        return total_get_qps + total_multi_get_qps + total_scan_qps;
-    }
-
-    double get_total_write_qps() const
```

Comment:
Function `get_total_write_qps` in `command_helper.h` doesn't add incr and duplicate qps